vm/src/stdlib/builtins.rs

```diff
@@ -367,7 +365,13 @@ mod builtins {
         obj.hash(vm)
     }
 
-    // builtin_help
+    #[pyfunction]
+    fn breakpoint(args: FuncArgs, vm: &VirtualMachine) -> PyResult {
+        let hook =
+            sys::get_object("breakpointhook".to_owned(), vm).expect("lost sys.breakpointhook");
```

Comment:
The `lost sys.breakpointhook` error is not a panic but a `RuntimeError` in CPython.